### ISSUE TITLE
Wide formatted sections after a column break are displayed incorrectly when exported to PDF

### ISSUE THREAD

**Issue by GE0NE** [naturalcrit/homebrewery#2185]
## Description

**Browser(s):** Chrome Version 101.0.4951.67 (Official Build) (64-bit) **and** Chrome 101.0.4951.61 (App)

**Operating System:** Windows 10 (x64) **and** Android 11

**Legacy or v3 Renderer:** v3 Renderer

Sections formatted with {{wide}} after a column break are rendered at the bottom of the page when exported to PDF instead of where they are displayed when rendered in HTML.

Steps to reproduce:

1. Insert any text

2. Create a column break \column

3. insert any text after the column break

4. insert a section and format it with {{wide}}

5. export as PDF

Result: the exported PDF will look different from the HTML preview.

**Screenshots**:

[An example when rendered in-browser](https://i.imgur.com/4tzBlFW.png)

[The same example when exported to PDF](https://i.imgur.com/WM44dFU.png)

[An example when rendered in-browser on mobile](https://i.imgur.com/jYlsIM1.jpg)

[The same example when exported to PDF](https://i.imgur.com/k0ILhtm.jpg)

**Share Link** : https://homebrewery.naturalcrit.com/share/1yozCdU06IHsucVrK_fPng38or0oCHNxaQEABe461gIlc

**Brew code to reproduce** : <details><summary>Click to expand</summary>
```
## Lorem Ipsum
Lorem ipsum dolor sit amet, consectetur adipiscing elit. Aenean vehicula pellentesque viverra. Phasellus id iaculis metus. Cras vestibulum mattis ipsum, vitae rhoncus magna consectetur eu.

\column

Lorem ipsum dolor sit amet, consectetur adipiscing elit. Aenean vehicula pellentesque viverra. Phasellus id iaculis metus. Cras vestibulum mattis ipsum, vitae rhoncus magna consectetur eu.

{{wide
### The Problematic Section

**This section appears fine when rendered as HTML but when exported to PDF it is sent to the bottom of the page.**

Lorem ipsum dolor sit amet, consectetur adipiscing elit. Aenean vehicula pellentesque viverra. Phasellus id iaculis metus. Cras vestibulum mattis ipsum, vitae rhoncus magna consectetur eu.
}}
```
</details>

**Comment by calculuschild** [comment]
Yeah, this is a bug with Chrome. As a workaround, you would need to avoid manual column-breaks above the `{{wide` items. You can get essentially the same result by using colons `:::` on a line to create vertical spacing at the point of the column-break until it forces a natural break where you want it. The PDF should work fine in that case.

**Comment by calculuschild** [comment]
See also https://github.com/naturalcrit/homebrewery/issues/2083 which is caused by the same issue, but inside a monster stat block.

**Comment by calculuschild** [comment]
Closing. This was fixed with Chrome v108.
